### ISSUE TITLE
Document installing thriftrw modules

### ISSUE THREAD

**Issue by abhinav** [uber/tchannel-python#284]
For generated modules, we should document how people can install them into `sys.modules` so that they can import from them more easily.

**Comment by ximyu** [comment]
So after doing this I will be able to import a generated Thrift struct directly without importing the whole module first?

**Comment by abhinav** [comment]
Yes. This is only partially documented in the `thrift.load` document right now, but basically you can do:

    # foo/bar.thrift

    import sys
    from tchannel import thrift

    sys.modules[__name__] = thrift.load(...)

This replaces the module 'foo.bar' with the generated module. Then other modules can simply do,

    from foo.bar import MyService, SomeType

**Comment by ximyu** [comment]
This is amazing. Thanks!
